### PR TITLE
[FIX] *: review 'frontend' pages init (web_editor.base) 

### PR DESCRIPTION
**Pull request by Gorash**
Following https://github.com/odoo/odoo/pull/29775

### CODE REVIEW

File: addons/auth_password_policy_signup/static/src/js/signup_policy.js [hunk 1]
```diff
@@ -1,11 +1,10 @@
 odoo.define('auth_password_policy_signup.policy', function (require) {
 "use strict";
 
-var base = require('web_editor.base');
 var policy = require('auth_password_policy');
 var PasswordMeter = require('auth_password_policy.Meter');
 
-base.ready().then(function () {
+$(function () {
```
Comment (qsm-odoo) [inline addons/auth_password_policy_signup/static/src/js/signup_policy.js]: I would prefer requiring `web.dom_ready` ... but it is ok.

File: addons/survey/static/src/js/survey.js [hunk 2]
```diff
@@ -169,11 +168,10 @@ if(!the_form.length) {
         return ajax.loadJS(url);
     }
 
-    var ready_with_locale = $.when(base.ready(), load_locale());
     // datetimepicker use moment locale to display date format according to language
     // frontend does not load moment locale at all.
     // so wait until DOM ready with locale then init datetimepicker
-    ready_with_locale.then(function(){
+    load_locale().then(function(){
```
Comment (qsm-odoo) [inline addons/survey/static/src/js/survey.js]: Should still wait for dom ready here

File: addons/web_editor/views/editor.xml [hunk 3]
```diff
@@ -125,9 +125,6 @@
     <xpath expr="//link[last()]" position="after">
         <link rel="stylesheet" type="text/scss" href="/web_editor/static/src/scss/web_editor.common.scss"/> <!-- not in common because compiled differently with context -->
     </xpath>
-    <xpath expr="//script[last()]" position="after">
-        <script type="text/javascript" src="/web_editor/static/src/js/root_widget.js"/>
-    </xpath>
```
Comment (qsm-odoo) [inline addons/web_editor/views/editor.xml]: Should probably be in another PR

File: addons/web_unsplash/static/src/js/unsplash_beacon.js [hunk 4]
```diff
@@ -1,26 +1,24 @@
...
-            }
-        });
-    }
+var unsplash_images = [];
+_.each($('img[src*="/unsplash/"]'), function (img, index) {
+    // get image id from URL (`http://www.domain.com:1234/unsplash/xYdf5feoI/lion.jpg` -> `xYdf5feoI`)
+    unsplash_images.push(img.src.split('/unsplash/')[1].split('/')[0]);
 });
+if (unsplash_images.length) {
+    ajax.jsonRpc('/web_unsplash/get_app_id', 'call').then(function (appID) {
```
Comment (qsm-odoo) [inline addons/web_unsplash/static/src/js/unsplash_beacon.js]: Don't we need the session to be bound to make correct rpc ? Don't know...

File: addons/website/static/src/js/content/body_manager.js [hunk 5]
```diff
@@ -1,10 +1,10 @@
 odoo.define('web_editor.BodyManager', function (require) {
 'use strict';
 
-var weContext = require('web_editor.context');
-var rootWidget = require('web_editor.root_widget');
+var weContext = require('website.context');
```
Comment (qsm-odoo) [inline addons/website/static/src/js/content/body_manager.js]: we is for web_editor, rename the variable too I suppose ? Same everywhere

File: addons/website/static/src/js/content/root_widget.js [hunk 6]
```diff
@@ -1,4 +1,4 @@
-odoo.define('web_editor.root_widget', function (require) {
```
Comment (qsm-odoo) [inline addons/website/static/src/js/content/root_widget.js]: Make a move commit for this

File: addons/website/static/src/js/website.js [hunk 7]
```diff
@@ -1,7 +1,7 @@
 odoo.define('website.website', function (require) {
 'use strict';
 
-var weContext = require('web_editor.context');
```
Comment (qsm-odoo) [inline addons/website/static/src/js/website.js]: Actually website.website is not needed anymore and should be included directly in website.context as this was only in charge of extending the context stuff. I can do it if you want, but either remove all the context-related stuff in this PR or do all the needed changes :wink: Don't want this to be included in the same commit as web_editor.base.

File: addons/survey/static/src/js/survey.js (hunk 2 above)
Comment (Gorash) [reply addons/survey/static/src/js/survey.js]: https://github.com/odoo/odoo/pull/30409/files/148345e5815f6d49ab04a8a0d404bcabc5e0a757#diff-55a36bd725045e96a61a869da9487453L4

File: addons/web_editor/views/editor.xml (hunk 3 above)
Comment (Gorash) [reply addons/web_editor/views/editor.xml]: :+1:

File: addons/website/static/src/js/content/body_manager.js (hunk 5 above)
Comment (Gorash) [reply addons/website/static/src/js/content/body_manager.js]: :+1:

File: addons/survey/static/src/js/survey.js (hunk 2 above)
Comment (qsm-odoo) [reply addons/survey/static/src/js/survey.js]: Oh. Lol.

Comment (Gorash) [reply addons/survey/static/src/js/survey.js]: :laughing:

File: addons/web_unsplash/static/src/js/unsplash_beacon.js (hunk 4 above)
Comment (Gorash) [reply addons/web_unsplash/static/src/js/unsplash_beacon.js]: tested, work fine

File: addons/web_editor/static/src/js/root_widget.js [hunk 8]
```diff
@@ -1,167 +0,0 @@
-odoo.define('web_editor.root_widget', function (require) {
-'use strict';
-
```
Comment (qsm-odoo) [inline addons/web_editor/static/src/js/root_widget.js]: Fail

File: addons/website/static/src/js/content/body_manager.js (hunk 5 above)
Comment (qsm-odoo) [inline addons/website/static/src/js/content/body_manager.js]: Fail

File: addons/website/static/src/js/content/body_manager.js [hunk 9]
```diff
@@ -1,10 +1,10 @@
...
 var ServiceProviderMixin = require('web.ServiceProviderMixin');
 var session = require('web.session');
+var rootWidget = require('website.root_widget');
```
Comment (qsm-odoo) [inline addons/website/static/src/js/content/body_manager.js]: Fail

File: addons/website/static/src/js/content/context.js [hunk 10]
```diff
@@ -1,4 +1,4 @@
-odoo.define('web_editor.context', function (require) {
+odoo.define('website.context', function (require) {
```
Comment (qsm-odoo) [inline addons/website/static/src/js/content/context.js]: Fail

File: addons/auth_password_policy_signup/static/src/js/signup_policy.js [hunk 11]
```diff
@@ -1,25 +1,23 @@
 odoo.define('auth_password_policy_signup.policy', function (require) {
```
Comment (qsm-odoo) [inline addons/auth_password_policy_signup/static/src/js/signup_policy.js]: Commit message fail:
- 72 characters
- Wrong targeting , portal has changed (and should be explained)

File: addons/website/static/src/js/content/root_widget.js [hunk 12]
```diff
@@ -0,0 +1,167 @@
+odoo.define('web_editor.root_widget', function (require) {
+'use strict';
```
Comment (qsm-odoo) [inline addons/website/static/src/js/content/root_widget.js]: Fail move

File: addons/website/static/src/js/content/body_manager.js (hunk 9 above)
Comment (Gorash) [reply addons/website/static/src/js/content/body_manager.js]: ?

Comment (Gorash) [reply addons/website/static/src/js/content/body_manager.js]: ho... I squash the wrong commit ^^;

**Comment by qsm-odoo** [comment]
So in fact, only two commits: one about context and one about base
